src/services/itemLogin/entities/guest.ts

```diff
@@ -19,7 +19,7 @@ export class Guest extends Account {
   })
   itemLoginSchema: ItemLoginSchema;
 
-  @Column('simple-json', { nullable: false, default: '{"lang":"de"}' })
+  @Column('simple-json', { nullable: false, default: '{}' })
```

Comment:
Shouldn't this trigger a migration? I don't see any in this PR.

Reply:
It is because defaults value doesn't works in Database Layer with TypeORM's ChildEntity. That one limitation of this inheritance... I'm keeping it in case of migration.